code/modules/research/xenoarchaeology/tools/gearbelt.dm

```diff
@@ -11,16 +10,18 @@
 		/obj/item/device/beacon_locator,
 		/obj/item/device/radio/beacon,
 		/obj/item/device/gps,
+		/obj/item/device/radio,
 		/obj/item/device/measuring_tape,
 		/obj/item/device/flashlight,
 		/obj/item/pickaxe,
 		/obj/item/device/depth_scanner,
 		/obj/item/device/camera,
+		/obj/item/device/camera_film,
 		/obj/item/paper,
 		/obj/item/photo,
 		/obj/item/folder,
 		/obj/item/pen,
-		/obj/item/folder,
```

Comment:
Undocumented change

Reply:
That's not undocumented, it was a duplicate. You can see that the same item is two lines above it.